Modules/timemodule.c

```diff
@@ -1135,7 +1137,9 @@ time_tzset(PyObject *self, PyObject *unused)
         return NULL;
     }
 
+#ifndef MS_WINDOWS_NON_DESKTOP
     tzset();
+#endif
```

Comment:
Currently `time.tzset()` isn't implemented on Windows because `HAVE_WORKING_TZSET` isn't defined. I don't know why it's not. It should be implemented for desktop builds.